### PR TITLE
upload project proposal

### PR DESCRIPTION
**Pull request by haashiagh10**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: project/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Raman Spectroscopy , and High-Performance Liquid Chromatography(HPLC) are the way to get Raman spectroscpy data.To deal with the data , We want to detect the feature in the liqid by checking the Raman data.
+However , the machine to get the spectral data and the model background are not quite clear.
+After remove the noise and background in the data , we can implement some algorithm to analyze the data.
+
+# System Architecture
+The module should be compatible with NumPy arrays
+![image](https://github.com/haashiagh10/propsal/blob/main/flowchart.png)
+
+# API Description
+Ideally, it would be available on PyPI, and one can install it using pip.
```
Comment (yungyuc) [inline project/README.md:14]: It is not necessary to make it in PyPI.  It's not very straight-forward to do it with a project using C++.

**Review by yungyuc — CHANGES_REQUESTED** [review]
Please provide a name for your project and add it to the list in the `project` root directory.

File: project/README.md (hunk 1 above)
Comment (haashiagh10) [reply project/README.md:16]: Sorry professor , what means add to the list in project root directory. I have pull request to the project folder on github. But I cannot see the folder in project folder. Do i miss something?

**Comment by yungyuc** [comment]
You have this PR and #178 .  Which one is your proposal?  Please keep only one and let me know which I should review again.

**Comment by haashiagh10** [comment]
I delete #178 and this one is my proposal. Thanks professor

**Review by yungyuc — CHANGES_REQUESTED** [review]
Please add your project to the list in the readme file in the `project` root and leave a message when it's done.

**Comment by haashiagh10** [comment]
I update the list in readme file thanks professor

**Comment by 1am9trash** [comment]
@yungyuc Int this merge, the file position is wrongly put in project directory, not in personal directory. It may be fixed. Otherwise, the preview in [project](https://github.com/yungyuc/nsdhw_22au/tree/project/project) displays wrong README file.

**Pull request merged**